modules/scope3/rtd/module_test.go

```diff
@@ -40,7 +40,7 @@ func getTestEntrypointPayload(t *testing.T) hookstage.EntrypointPayload {
 func TestBuilder(t *testing.T) {
 	config := json.RawMessage(`{
 		"enabled": true,
-		"endpoint": "https://rtdp.scope3.com/amazonaps/rtii",
+		"endpoint": "https://rtdp.scope3.com/prebid/prebid",
```

Comment:
I see this string in many places - maybe you can have a constant in the non-test code (that you use for the default in the case where not set, and can also use here and in your assertion)?